src/plugins/console/server/config.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export type ConfigType = TypeOf<typeof config>;
+
+export const config = schema.object(
+  {
+    enabled: schema.boolean({ defaultValue: true }),
+    proxyFilter: schema.arrayOf(schema.string(), { defaultValue: ['.*'] }),
+    ssl: schema.object({ verify: schema.boolean({ defaultValue: false }) }, {}),
+
+    // This does not actually work, track this issue: https://github.com/elastic/kibana/issues/55576
```

Comment:
I'm guessing this issue blocks this PR?

Reply:
I don't think it needs to block this PR, but it would be nice to do the testing here if we can :). Otherwise, I can do it in post-merging too.

Reply:
Has been solved already! 😄 Thanks @pgayvallet!